docs/en/ingest-management/elastic-agent/scaling-on-kubernetes.asciidoc

```diff
@@ -0,0 +1,272 @@
+
+- The amount of metrics being collected from several kubernetes endpoints
+- The agent's resources to cope up with the higher CPU and Memory needed for internal processing
+- The ElasticSearch resources due to higher rate of metric ingestion needs
+- The Dashboard's Visualisations response times as more data are requested on a given time window 
+
+The document is divided in two main sections:
+
+- <<configuration-practices,Configuration practices>>
+- <<validation-and-troubleshooting-practices,Validation and Troubleshooting practices>>
+
+[discrete]
+[[configuration-practices]]
+== Configuration Practices
+
+[discrete]
+=== Configure Agent Resources
+
+The kubernetes observability is based on https://docs.elastic.co/en/integrations/kubernetes[kubernetes integration], which is fetching metrics from several components:
+
+- **Per node:**
+  - kubelet
+  - controller-manager
+  - scheduler
+  - proxy
+- **Cluster Wide (i.e. unique metrics for the whole cluster):**
+  - kube-state-metrics
+  - apiserver
+
+The Elastic Agent manifest is deployed by default as daemonset. That said, each elastic agent by default is being deployed on every node of the Kubernetes cluster. 
+
+Additionally, by default one agent is elected as **leader**(for more information visit <<kubernetes_leaderelection-provider>>). This is configured in our https://github.com/elastic/elastic-agent/blob/main/deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml#L32[manifests]. The Agent Pod which holds the leadership lock will be responsible for also collecting the cluster wide metrics in addition to its node's metrics.
+
+--
+[role="screenshot"]
+image::../images/k8sscaling.png[Agent as daemonset]
+--
+
+The above schema explains how {agent} collects and sends metrics to Elasticsearch. Because of Leader Agent being responsible to also collecting cluster-lever metrics, this means that it requires additional resources.
+
+Daemonset deployment approach simplifies the installation of the {agent} because we define less Kubernetes Resources in our manifest and we only need one single Agent policy for our Agents. Hence it is the default supported method for <<running-on-kubernetes-managed-by-fleet, Managed Elastic Agent installation>>
+
+
+[discrete]
+=== Specifying resources and limits in Agent manifests
+
+Based on our tests we advise to configure only the `limit` section of the `resources` section in the manifest. In this way the `request`'s settings of the `resources` will fall back to the `limits` specified. The `limits` is the upper bound limit of your microservice process, meaning that can operate in less resources and protect Kubernetes to assign bigger usage and protect from possible resource exhaustion.
```

Comment:
This is a bit hard to relate in a doc like this so instead I think we should say there are the things you need to consider: resourcing and the scheduling priority and maybe mention what can go wrong if you under-resource or you set the wrong priority.

Reply:
I have updated @pjbertels . Now?